MetaMask/metamask-mobile
# [Mobile Snaps] Snaps controllers v3.2.0

## **Description**
Update snaps controller, snaps utils and snaps-rpc-methods to version 3.2.0

See snaps [changelog](https://github.com/MetaMask/snaps/blob/main/packages/snaps-controllers/CHANGELOG.md)

I followed the guidance of these extension PRs
- https://github.com/MetaMask/metamask-extension/pull/21612
- https://github.com/MetaMask/metamask-extension/pull/21255

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/116

## **Manual testing steps**

build branch
run yarn setup
navigate to browser tab
go to http://metamask.github.io/snaps/test-snaps/1.1.0
install BIP-32 snap
go through approval flow
the snap should be installed
navigate to settings/snaps
the snap should be available in settings
you should be able to preview all of the snaps permissions in the snap settings

## **Screenshots/Recordings**




https://github.com/MetaMask/metamask-mobile/assets/22918444/07444f5c-9dee-481f-afa5-3150cc60dc84



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
